boot/freeldr/freeldr/arch/uefi/ueficon.c

```diff
@@ -16,6 +16,8 @@ extern EFI_SYSTEM_TABLE* GlobalSystemTable;
 static unsigned CurrentCursorX = 0;
 static unsigned CurrentCursorY = 0;
 static unsigned CurrentAttr = 0x0f;
+static EFI_INPUT_KEY Key;
+static BOOLEAN scancode;
```

Comment:
Unused vars
```
../src/boot/freeldr/freeldr/arch/uefi/ueficon.c:20:16: error: 'scancode' defined but not used [-Werror=unused-variable]
 static BOOLEAN scancode;
                ^~~~~~~~
../src/boot/freeldr/freeldr/arch/uefi/ueficon.c:19:22: error: 'Key' defined but not used [-Werror=unused-variable]
 static EFI_INPUT_KEY Key;
                      ^~~
```

Reply:
yeah sorry! Im currently playing with this, with the other 2 architectures on top of it locally right now. I'm trying to make sure what I'm going to do will work across the board as is

This will be pushed with pretty large changes to it asap.